### PR TITLE
RDM-6193-PG-Persis

### PR DESCRIPTION
**Pull request by nathan-clark**
### Change description ###
Turn Pg persistence on. 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Comment by nathan-clark** [comment]
> @nathan-clark I assume there's a PR ready as well to remove this setting from the flux config?

yes was awaiting approval, will check with Anand on how we test the flux config

File: ccd/values.yaml [hunk 1]
```diff
@@ -252,7 +252,7 @@ dm-store:
 
 postgresql:
   persistence:
-    enabled: false
+    enabled: true
```
Comment (timja) [inline ccd/values.yaml]: this is a risky default, what is the reason for this change?

Basically chart-ccd PRs normally get a lot more unstable with this change, at the minimum it should be disabled in ci-values.yaml

and ccd-per-pr shouldn't have this enabled either

Comment (mario-paniccia) [reply ccd/values.yaml]: @timja why persistency causes unstableness? 
in that case we could disable this on ci-values.yaml sure. 
But it seems normal to me to assume that CCD on Demo or any other env where the product chart is used should be persistent. I assume the product chart will be used for testing etc, and losing test data during the test process it's painful.

Comment (timja) [reply ccd/values.yaml]: There's a limit of ~15 disks attached to a node, they don't detach very well under high load and are a cause of flakiness on preview and instability to the cluster.

we currently have a blanket rule on no persistent volumes on preview.

Comment (mario-paniccia) [reply ccd/values.yaml]: @timja does this apply to Demo as well other than Preview? In that case we will revert this change. But if the issue does not apply to Demo, I'd rather keep persistency enabled by default. I don't want Services opening investigation tickets to CCD when data disappears on the env

Comment (mario-paniccia) [reply ccd/values.yaml]: you mean on Preview other teams will use chart-ccd?

**Comment by nathan-clark** [comment]
will redo with comments

**Pull request closed without merging**